src/wazuh_modules/vulnerability_scanner/tests/policyManager/policyManager_test.cpp

```diff
@@ -17,65 +17,121 @@ void PolicyManagerTest::SetUp()
 
 void PolicyManagerTest::TearDown() {};
 
-TEST_F(PolicyManagerTest, validConfigurationVulnerabilityDetection)
+TEST_F(PolicyManagerTest, invalidConfigurationNoIndexer)
 {
-    const auto& configJson {nlohmann::json::parse(R"({
-    "enabled": "yes",
-    "index-status": "yes",
-    "feed-update-interval": "60m",
-    "offline-url": "file:///var/algo.tar.gz"
-})")};
-    EXPECT_NO_THROW(m_policyManager->validateAndConfigureVulnerabilityDetection(configJson));
+    // Configuration with missing "indexer" field.
+    const auto& configJson = R"(
+      {
+        "vulnerability-detection":
+        {
+          "dummy": true
+        }
+      }
+    )"_json;
 
-    EXPECT_TRUE(m_policyManager->isVulnerabilityDetectionEnabled());
-    EXPECT_TRUE(m_policyManager->isIndexerEnabled());
+    ASSERT_NO_THROW(m_policyManager->loadConfiguration(configJson));
+}
 
-    EXPECT_STREQ(m_policyManager->getFeedUrl().c_str(), "file:///var/algo.tar.gz");
-    EXPECT_EQ(m_policyManager->getFeedUpdateTime(), 3600);
+TEST_F(PolicyManagerTest, invalidConfigurationNoVulnerabilityDetection)
+{
+    // Configuration with missing "vulnerability-detection" field.
+    const auto& configJson = R"(
+      {
+        "indexer":
+        {
+          "dummy": true
+        }
+      }
+    )"_json;
+
+    ASSERT_NO_THROW(m_policyManager->loadConfiguration(configJson));
```

Comment:
The method `loadConfiguration()` simply copies a variable and performs no further operations.
It isn't testing anything.